x-pack/plugins/upgrade_assistant/public/application/components/es_deprecation_logs/fix_deprecation_logs/fix_logs_step.tsx

```diff
@@ -93,16 +88,11 @@ const i18nTexts = {
 };
 
 interface Props {
-  setIsComplete: OverviewStepProps['setIsComplete'];
   hasPrivileges: boolean;
   privilegesMissing: MissingPrivileges;
 }
 
-const FixLogsStep: FunctionComponent<Props> = ({
-  setIsComplete,
-  hasPrivileges,
-  privilegesMissing,
-}) => {
+const FixLogsStep: FunctionComponent<Props> = ({ hasPrivileges, privilegesMissing }) => {
```

Comment:
TODO: We should rename this function to something like `FixDeprecationLogs`, and the same goes for the file itself